compose/ui/ui-text/src/jsWasmMain/kotlin/androidx/compose/ui/text/intl/JsPlatformLocale.js.kt

```diff
@@ -16,29 +16,51 @@
 
 package androidx.compose.ui.text.intl
 
-//TODO: Implement JsLocale
-internal class JsLocale(val locale: Any) : PlatformLocale {
+internal class JsLocale(val locale: IntlLocale) : PlatformLocale {
+
+    constructor(languageTag: String): this(languageTag.toIntlLocale())
+
     override val language: String
-        get() = ""//locale.languageCode!!
+        get() = locale.language
 
     override val script: String
-        get() = ""//locale.scriptCode!!
+        get() = locale.script ?: ""
 
     override val region: String
-        get() = ""//locale.countryCode!!
+        get() = locale.region ?: ""
 
-    override fun toLanguageTag(): String = TODO("implement native toLanguageTag") // locale.toLanguageTag()
+    override fun toLanguageTag(): String = locale.baseName
 }
 
 internal actual fun createPlatformLocaleDelegate(): PlatformLocaleDelegate =
     object : PlatformLocaleDelegate {
         override val current: LocaleList
-            get() = LocaleList(listOf(Locale(JsLocale(Any()))))
+            get() = LocaleList(
+                listOf(Locale(JsLocale(currentLanguageTag())))
```

Comment:
We're dealing with list here, so let's use [`navigator.languages`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language) instead

Reply:
fixed